kyuubi-zookeeper/src/main/scala/org/apache/kyuubi/zookeeper/ZookeeperConf.scala

```diff
@@ -87,26 +75,26 @@ object ZookeeperConf {
       .intConf
       .createWithDefault(120)
 
-  val ZK_MIN_SESSION_TIMEOUT: OptionalConfigEntry[Int] =
+  val ZK_MIN_SESSION_TIMEOUT: ConfigEntry[Int] =
     buildConf("zookeeper.embedded.min.session.timeout")
       .doc("minSessionTimeout in milliseconds for the embedded zookeeper server will allow the" +
         " client to negotiate. Defaults to 2 times the tickTime")
       .version("1.2.0")
       .intConf
-      .createOptional
+      .createWithDefault(3000 * 2)
 
-  val ZK_MAX_SESSION_TIMEOUT: OptionalConfigEntry[Int] =
+  val ZK_MAX_SESSION_TIMEOUT: ConfigEntry[Int] =
     buildConf("zookeeper.embedded.max.session.timeout")
       .doc("maxSessionTimeout in milliseconds for the embedded zookeeper server will allow the" +
         " client to negotiate. Defaults to 20 times the tickTime")
       .version("1.2.0")
       .intConf
-      .createOptional
+      .createWithDefault(3000 * 20)
 
-  val ZK_SERVER_ID: ConfigEntry[Int] =
```

Comment:
This is a breaking change, let mark it deprecated and useless first, then remove it after v1.3.0 release.

Reply:
Got.